CONTRIBUTING.md

```diff
@@ -44,6 +44,25 @@ Build logs are written to stdout and artifacts written to the current directory.
 When new versions of the packages are released, they should be included in [CHaP](https://github.com/input-output-hk/cardano-haskell-packages).
 See the CHaP README for [instructions](https://github.com/input-output-hk/cardano-haskell-packages#-from-github).
 
+## Release Branches
+
+When needed we use release branches: `release/cardano-node-*`, e.g.
+`release/cardano-node-1.35.x` (which ought to be simply named as
```

Comment:
I don't get the relationship between `release/cardano-node-1.35.x` and `release/cardano-node-1.35`, ie what are their types :smile: